test/controllers/api/rhsm/candlepin_proxies_controller_test.rb

```diff
@@ -113,7 +113,10 @@ module Katello
         System.stubs(:where).returns(@system)
         System.any_instance.stubs(:first).returns(@system)
         uuid = @system.uuid
-        User.stubs(:current).returns(CpConsumerUser.new(:uuid => uuid, :login => uuid))
```

Comment:
@unorthodoxgeek because `attr_accesible` is being removed in foreman 1.11. It's a short-term solution in the rails world.